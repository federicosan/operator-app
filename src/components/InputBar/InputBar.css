.InputBar {
  display: flex;
  background: #fff;
  min-height: 50px;
}

.InputBar textarea {
  flex-grow: 1;
  outline: 0;
  display: flex;
  padding: 1rem;
  resize: none;
  border: none;
}

.InputBar .Button--send {
  margin: 4px;
}
