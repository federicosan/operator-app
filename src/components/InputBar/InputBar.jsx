import React, { Component } from 'react';
import { connect } from 'react-redux';
import './InputBar.css'
// import PanelBar from '../PanelBar/PanelBar.jsx';

export class InputBarComponent extends Component {
  render () {
    return (
      <div className="InputBar">
        <textarea placeholder="text here, dummy" />
      </div>
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({

});

const InputBar = connect(
  mapStateToProps,
  mapDispatchToProps,
)(InputBarComponent);

export default InputBar;
