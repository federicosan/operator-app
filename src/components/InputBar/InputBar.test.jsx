import React from 'react';

import { shallow } from 'enzyme';

import InputBar from './InputBar.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('InputBar', () => {
  it('matches snapshot', () => {
    const component = shallow(<InputBar store={store} />);

    expect(component).toMatchSnapshot();
  });
});
