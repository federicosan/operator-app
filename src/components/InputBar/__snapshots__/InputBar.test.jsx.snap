exports[`InputBar matches snapshot 1`] = `
<InputBarComponent
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
